reana_client/cli/ping.py

```diff
@@ -43,9 +43,13 @@ def ping(ctx, access_token):  # noqa: D301
         click.echo(
             click.style(
                 "REANA server: {0}\n"
-                "Authenticated as: {1} <{2}>\n"
-                "Status: {3}".format(
+                "REANA server version: {1}\n"
+                "REANA client version: {2}\n"
+                "Authenticated as: {3} <{4}>\n"
+                "Status: {5}".format(
                     get_api_url(),
+                    response.get("reana_server_version"),
```

Comment:
What if server does not contain `reana_server_version`?  Note that this PR must make sure that newly released reana-client 0.7.1 would work with previously released REANA cluster 0.7.0 which may not contain the new field. So we have to be very liberal in what server may send.  Kind of:

```
if response contains reana_server_version:
then
   print reana_server_version
else
  do nothing
endif
```

P.S. Please also rebase against latest `maint-0.7`.

Reply:
Done 👍